Base/Logger/Logger.Static/FileLogger.cpp

```diff
@@ -221,3 +221,9 @@ void FileLogger::debug(const std::string& message)
 {
     log(message, LogLevel::DEBUG);
 }
+
+void FileLogger::logAndThrow(const char* filename, std::uint16_t line, std::exception* exception, LogLevel severity)
+{
+    log('"' + std::string(exception->what()) + "\" on line " + std::to_string(line) + " in the file " + filename, severity);
+    throw exception;
```

Comment:
Here you throw a pointer, not an object. 
Not sure that this is correct.

Reply:
It's not matter now.
Can we resolve this conversation?

Reply:
Yep. Looks working